ITI/index.html

```diff
@@ -92,13 +125,14 @@ <h2 id="1.3">Public Comment</h2>
 							</div>
 							<div class="callout">
 								<h2 id="1.4">Supplements for Trial Implementation</h2>
+								<p>Submit comments and implementer feedback <a href="https://www.ihe.net/ITI_Public_Comments/">here</a>. Additionally, for documents maintained in the IHE GitHub Repository, a link to submit GitHub issues is included following the document name in the list below.</p>
 								<p>The IHE IT Infrastructure Technical Committee invites organizations to begin development work based on the following supplements to the IT Infrastructure Technical Framework. These trial implementation profiles are eligible for testing at subsequent IHE Connectathons.</p>
 
 								<ul>
 									<li><a title="Add RESTful Query to ATNA" href="https://www.ihe.net/uploadedFiles/Documents/ITI/IHE_ITI_Suppl_RESTful-ATNA.pdf">Add RESTful ATNA (Query and Feed)</a> - Revised 2023-08-04</li>
 									<li><a title="Advanced Patient Privacy Consents (APPC)" href="https://www.ihe.net/uploadedFiles/Documents/ITI/IHE_ITI_Suppl_APPC.pdf">Advanced Patient Privacy Consents (APPC)</a> - Revised 2021-07-02</li>
 									<li><a title="Asynchronous AS4 Option" href="https://www.ihe.net/uploadedFiles/Documents/ITI/IHE_ITI_Suppl_AsyncAS4Option.pdf" target="_blank" rel="noopener noreferrer">Asynchronous AS4 Option</a> - Revised 2023-08-04</li>
-									<li><a title="Basic Audit Log Patterns (BALP)" href="./BALP/index.html">Basic Audit Log Patterns (BALP) Release 1.1.2</a> - Revised 2023-08-04  <i class="fi-web" title="Web Publication"></i><i class="fi-star" title="Implementation Guide"></i></li>
+									<li><a title="Basic Audit Log Patterns (BALP)" href="./BALP/index.html">Basic Audit Log Patterns (BALP) Release 1.1.2</a> - Revised 2023-08-04  <i class="fi-web" title="Web Publication"></i><i class="fi-star" title="Implementation Guide"></i>   (Create a <a href="https://github.com/IHE/ITI.BasicAudit/issues/new/choose">GitHub Issue</a>)</li>
```

Comment:
why are you doing this? All of the rest of ITI IG profiles don't have this, they rely on the footer in the IG.